api.py

```diff
@@ -36,22 +36,22 @@ def open_pieces_os() -> Optional[str]:
 	version = get_version()
 	if version:
 		return version
-	# sublime.platform() ->  Literal['osx', 'linux', 'windows']
-	pl = sublime.platform().lower()
-	if pl == "windows":
-		subprocess.Popen(["start", "os_server"], shell=True)
-	elif pl == "linux":
-		subprocess.Popen(["os_server"])
-	elif pl == "osx":
-		subprocess.Popen(["open", "os_server"])
-	# Check pieces every 2 seconds if it is opened
-	for _ in range(4):
-		time.sleep(2) # wait for the server to open
-		version = get_version()
-		if version:
-			return version
-	return get_version() # pieces os version
-
+	else:
```

Comment:
we don't actually need this else statement here if we are going to return in the above if statement

Reply:
Yupp true that will be more readable